spec/fixtures/schema.rb

```diff
@@ -46,26 +53,6 @@
     query QueryType
     mutation MutationType
```

Comment:
@DamirSvrtan I know why. I forgot to add `use BatchLoader::GraphQL` in the GraphQL 1.7 schema in this PR https://github.com/exAspArk/graphql-errors/pull/13, my bad

Reply:
Brilliant, thanks for the help! I'll add it!

Reply:
One interesting thought dough - if you forgot to add `use BatchLoader::GraphQL`, then it was resolved as a regular field, right? Shouldn't it then be caught by the normal field resolve proc?

Reply:
Yeah, good question. It should but, unfortunately, it fails somewhere on the GraphQL-Ruby level when it tries to detect whether the object is lazy or not. And when it performs the check, `BatchLoader` is being resolved with an exception somewhere outside GraphQL instrumenters